tests/core/baseline_test.py

```diff
@@ -80,18 +80,24 @@ def test_scan_all_files():
             assert get_relative_path_if_in_cwd(f.name) in secrets.data
 
 
-@pytest.mark.xfail(
-    reason=(
-        'TODO: When we\'re done with changes, we\'ll update .secrets.baseline and this should '
-        'work then.'
-    ),
-)
 def test_load_and_output():
     with open('.secrets.baseline') as f:
         data = json.loads(f.read())
 
-    secrets = baseline.load(data)
-    assert baseline.format_for_output(secrets) == data
+    secrets = baseline.load(data, filename='.secrets.baseline')
+    output = baseline.format_for_output(secrets)
+
+    for item in [data, output]:
+        item.pop('generated_at')
```

Comment:
FWIW I think this does point out a potential problem, which is you're always going to have a merge-conflict due to `generated_at` if two branches both touch the baseline file.